migration/src/lib.rs

```diff
@@ -71,6 +74,9 @@ impl MigratorTrait for Migrator {
             Box::new(m20230919_072154_cl_audits::Migration),
             Box::new(m20231019_120101_add_seq_numbers_bgum_update_metadata::Migration),
             Box::new(m20231206_120101_remove_was_decompressed::Migration),
+            Box::new(m20240104_203133_add_cl_audits_v2::Migration),
+            Box::new(m20240104_203328_remove_cl_audits::Migration),
```

Comment:
The reason to use a different table is so that providers (like ourselves) can make the change with zero downtime. You can run both in parallel if you need to.

Note that the new table stores all data as "bytea" and uses an enum to reduce storage. These changes reduced our table size from about 800GB to 46GB!

Reply:
Ah I did not notice that but that is significant!!!